src/app/App.js

```diff
@@ -1,4 +1,24 @@
-export const App = ({options}) => {
-}
+export const App = ({ options }) => {
+  const buttonRanking = document.querySelector('.button--ranking');
 
+  function switchToHallOfFame() {
+    buttonRanking.lastElementChild.innerText = 'Rules';
+    buttonRanking.firstElementChild.classList = 'fas fa-graduation-cap';
```

Comment:
jeśli z jakiegoś powodu zmieni się wygląd HTMLa to kod gdzie operujemy na first/lastElementChild przestanie działać. moim zdaniem lepiej dodać klasy do poszczególnych elementów i wyciągnąć je do JSa przez querySelector('.class') żeby uniknąć ewentualnych problemów później. 
Dodatkowym plusem jest czytelność, teraz musiałem wejść w HTML i szukać do czego to się odnosi :)

Reply:
sto procent się zgadzam